src/components/search/search.component.tsx

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import {
+  Platform,
+  TextInput,
+  StyleSheet,
+  Dimensions,
+  View
+} from "react-native";
+import { SimpleLineIcons, Feather, FontAwesome } from "@expo/vector-icons";
+import Autocomplete from "./autocomplete.component";
+import { POI } from "../../types/main";
+import SeparatorSVG from "../../../assets/line-separator.svg";
+import { getSearchBarAutoCompleteHeight } from "../../services/autoCompleteHeight.service";
+
+/**
+ * the name and types of the properties types accepted
+ * by the Search component
+ */
+interface IProps {
+  setDestination: (poi: POI) => void;
+  queryText: (
+    userInput: string,
+    setAState: (poi: POI[]) => void,
+    onChange: (string: string) => void
+  ) => void;
+  setUserLocation: () => void;
+}
+
+/**
+ * Search component for indoor points of interest's
+ * @param getDestination Function called to update destination
+ * @param queryText Function to query POI based on user input
+ */
+const Search = ({ setDestination, queryText, setUserLocation }: IProps) => {
+  const [value, onChangeText] = React.useState("");
+  const [autoCompleteValues, setAutocomplete] = React.useState(null);
+
+  return (
+    <View
+      testID="searchContainer"
+      style={[
+        styles.container,
+        { height: getSearchBarAutoCompleteHeight(autoCompleteValues) }
+      ]}
+    >
+      <View style={styles.parent}>
+        <View style={styles.view}>
+          <View style={styles.view}>
+            <SimpleLineIcons name="menu" color="#AA2B45" size={26} />
+            <SeparatorSVG style={styles.lineSeperator} />
```

Comment:
Thanks for the SVGs! 😅

Reply:
So for omnibox-directions.component I've kept the PNGs but I used higher res images and scaled them down to keep the image quality. Had to do this because snapshots were failing with svg's for that component only and I was unable to find a fix at the moment.